src/openforms/submissions/models/submission.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
Why is this needed?

Reply:
To avoid having to use forward references in type hints if imported in an `if TYPE_CHECKING` block:

`def registrator(self) -> "AuthInfo | RegistratorInfo | None":` -> `def registrator(self) -> AuthInfo | RegistratorInfo | None:`

(`from __future__ import annotations` will make all type annotations as forward string references by default)